script/Arcueid/Day 4/04_00_ARC04_6_2B.txt

```diff
@@ -667,12 +696,14 @@ Arcueid is proudly holding her finger aloft.
 [sha:b0d387bf5189c16824f07d4d420a275c6ac75a15]{
 -- Page 521, Offset 6992.
 -- だから基礎編。だいじょうぶ、先輩としてバシッと教えてあげるから」
-That's why it's called 'Vampire Fundamentals'. Don't worry, as your senpai I'll be sure to teach you properly."
+Hence: 'Vampire Fundamentals'. Don't worry, Senpai over here's gonna give you a quick lesson."
+//AHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH /petrikow
 }
 [sha:f72bc38f7e61b6edcf6f9c36f2c44a8c113eabc2]{
 -- Page 522, Offset 6993.
 -- 誰が先輩か。……というか、もう基礎編とやらを始めてるよな、コイツ……。
-Who are you calling senpai? ...And besides, you already started explaining the fundamentals. This woman...
+Senpai in what... Wait, she's already getting started with her whole 'Fundamentals' shtick.
```

Comment:
I thought this was more like "Besides, if this is the fundamentals, what were we talking about before?" kinda dealio